<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mixture</class>
 <widget class="QWidget" name="Mixture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1568</width>
    <height>973</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,2,1">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBoxRecipeSelection">
           <property name="title">
            <string>Recipe info</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="labelSelectRecipe">
              <property name="text">
               <string>Recipe</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBoxRecipe"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelBasalt">
              <property name="text">
               <string>Basalt 0-8 (kg)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditBasalt">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>150</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelSand">
              <property name="text">
               <string>Sand 0-2 (kg)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEditSand">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>20</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelCement">
              <property name="text">
               <string>Cement (kg)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEditCement">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelWater">
              <property name="text">
               <string>Water (kg)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEditWater">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labelPlastifier">
              <property name="text">
               <string>Plastifier (kg)</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEditPlastifier">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="labelPigment">
              <property name="text">
               <string>Pigment (kg)</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="lineEditPigment">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QPushButton" name="pushButtonStartBatch">
              <property name="text">
               <string>Start batch</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxMixerWork">
           <property name="title">
            <string>Mixer work</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="labelMixerWork">
              <property name="text">
               <string>Mixer work description</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEditMixerWork">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="plainText">
               <string>Dry mix work = 10 kJ per kg
Wet mix work = 5 kJ per kg
Viscosity power  set at 1.2 kW
Plastifier mix work = 15 kJ per kg
Viscosity power final set at 1 kW

All above targets have a 1% error margin</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxBunker">
           <property name="title">
            <string>Remaing material in bunker</string>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="labelBasaltRemaining">
              <property name="text">
               <string>Basalt material remaining</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QProgressBar" name="progressBarBasalt">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelSandRemaining">
              <property name="text">
               <string>Sand material remaining</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QProgressBar" name="progressBarSand">
              <property name="value">
               <number>75</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBoxScaleBin">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Scale bin</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButtonTareScaleBin">
              <property name="text">
               <string>Tare scale bin</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButtonCalibrateScaleBin">
              <property name="text">
               <string>Calibrate scale bin</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLCDNumber" name="lcdNumberScaleBin">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxWater">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Scale water</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButtonTareScaleWater">
              <property name="text">
               <string>Tare scale water</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButtonCalibrateScaleWater">
              <property name="text">
               <string>Calibrate scale water</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLCDNumber" name="lcdNumberScaleWater">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Pigment</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButtonTareScalePigment">
              <property name="text">
               <string>Tare scale pigment</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButtonCalibrateScalePigment">
              <property name="text">
               <string>Calibrate scale pigment</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLCDNumber" name="lcdNumber_2"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxScalePlastifier">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Scale plastifier</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButtonTareScalePlastifier">
              <property name="text">
               <string>Tare scale plastifier</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButtonCalibrateScalePlastifier">
              <property name="text">
               <string>Calibrate scale plastifier</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLCDNumber" name="lcdNumberScalePlastifier"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxBinMaterial">
         <property name="title">
          <string>Bin material</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="labelIntegrationFactor">
            <property name="text">
             <string>Using integration factor: N</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QPushButton" name="pushButtonDoseNPulses">
            <property name="text">
             <string>Dose N pulses</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditPulseCount"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelPulseCount">
            <property name="text">
             <string>Set pulsecount</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButtonStartDosing">
            <property name="text">
             <string>Start dosing materials</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="5">
           <widget class="QTableWidget" name="tableWidgetBinR">
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Material</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Pulsecount</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Weight</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>Basalt</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>900</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>94</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>Basalt</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>100</string>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string>6</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QLCDNumber" name="lcdNumberBinMaterial">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="3" colspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Recipe progression</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QLCDNumber" name="lcdNumberRecipeProgression">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListWidget" name="listWidgetRecipeItems"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
